aea/crypto/ledger_apis.py

```diff
@@ -156,17 +155,16 @@ def transfer(self, identifier: str, crypto_object: Crypto, destination_address:
                     time.sleep(3.0)
 
             return tx_digest
-        else:
+        else:                   # pragma: no cover
             tx_digest = None
         return tx_digest
 
-    def is_tx_settled(self, identifier: str, tx_digest: str, amount: int) -> bool:
```

Comment:
Please add the amount back in. We will require this next week when the ledger api will be updated.